ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -310,8 +334,6 @@ def triangular(left, mode, right, size=None):
     return ivy.where(condition, values1, values2)
 
 
-@to_ivy_arrays_and_back
```

Comment:
could you please revert this change and also the changes to the array api submodule? Thanks @Honeson 😄
(would be great if you could take it forward with the PR @NripeshN, thanks!)

Reply:
Hi @vedpatwardhan 

I have made the changes you requested and created a new PR to keep things cleaner. Here is a link to the new PR for your review: https://github.com/unifyai/ivy/pull/27140

I will appreciate your feedback also @NripeshN.

Thank you.